buildtest/cli/__init__.py

```diff
@@ -925,10 +925,14 @@ def config_menu(subparsers):
         help="Specify a list of directories to search for modules via MODULEPATH to detect compilers",
     )
 
-    subparsers_compiler.add_parser(
+    compiler_test = subparsers_compiler.add_parser(
         "test",
         help="Test each compiler instance by performing module load test",
     )
+    compiler_test.add_argument(
```

Comment:
can you add a help message something like `help="Specify a compiler name to test by running module test"`